sdk/storage/azure-storage-blobs/src/blob_service_client.cpp

```diff
@@ -263,4 +263,22 @@ namespace Azure { namespace Storage { namespace Blobs {
         std::move(blobContainerClient), std::move(response.RawResponse));
   }
 
+  Azure::Response<BlobContainerClient> BlobServiceClient::RenameBlobContainer(
+      const std::string& sourceBlobContainerName,
```

Comment:
I think using pass by value here without const, and move-assign it to protocolLayerOptions's corresponding field makes sense and might save an additional copy as well. As you mentioned in a previous chat with Victor.

Reply:
Yeah, but that's a very subtle difference